src/coins.cpp

```diff
@@ -102,11 +102,11 @@ CCoinsMap::const_iterator CCoinsViewCache::FetchCoins(const uint256 &txid) const
 }
 
 
-bool CCoinsViewCache::GetAnchorAt(const uint256 &rt, libzerocash::IncrementalMerkleTree &tree) const {
+bool CCoinsViewCache::GetAnchorAt(const uint256 &rt, ZCIncrementalMerkleTree &tree) const {
     CAnchorsMap::const_iterator it = cacheAnchors.find(rt);
     if (it != cacheAnchors.end()) {
         if (it->second.entered) {
-            tree.setTo(it->second.tree);
+            tree = it->second.tree;
```

Comment:
This is using an implicit copy constructor, which I think is fine (but I didn't bother to look up any c++ standard to see what the implicit copy constructor does).